.gitignore

```diff
@@ -1 +1,3 @@
-my_*.cfg
\ No newline at end of file
+my_*.cfg
+out/
+.config
```

Comment:
Perhaps we should also add ignores for `*.out/` and `*.config`? That way I can easily use other paths without them "dirtying" the repo. I've currently got a couple mmu type units that i switch between, so being able to have `bt.out` and `bt.config` for my box turtle and `tr.out` and `tr.config` for my tradrack would be nice. I do this for building klipper on my mcu's so I don't have to re-configure the mcu each time I build and flash.